src/components/Other/Team.vue

```diff
@@ -144,4 +144,7 @@ export default {
   color: black;
   text-align: center;
 }
+.contact {
+  font-size: 35px;
```

Comment:
That's quite large. What if we don't show the no contact panel altogether

Reply:
yes, that would also be okay.